types/expo-localization/index.d.ts

```diff
@@ -0,0 +1,73 @@
+// Type definitions for expo-localization 1.0
+// Project: https://docs.expo.io/
+// Definitions by: Bartosz Dotryw <https://github.com/burtek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface LocalizationProps {
+    /**
+     * Native device language, returned in standard format. ex: `en-US`, `es-US`
+     */
+    locale: string;
+
+    /**
+     * List of all the native languages provided by the user settings. These are returned in the order the user defines in their native settings
+     */
+    locales: string[];
+
+    /**
+     * Country code for your device
+     */
+    country: string | undefined;
```

Comment:
Based on the documentation this property is a Flow `maybe` type. Meaning it's `string | undefined | null`